test/cypress/integration/login_spec.ts

```diff
@@ -94,14 +94,58 @@ describe('Login', () => {
         cy.contains('Unfortunately you cannot login since your account is not associated with any tenants.');
     });
 
-    it('fails when provided credentials are invalid', () => {
-        cy.login({ password: 'invalid-password', expectSuccessfulLogin: false });
-
-        cy.get('.error.message').should(
-            'have.text',
-            'User unauthorized: Authentication failed for user admin. Wrong credentials or locked account'
-        );
-        cy.location('pathname').should('be.equal', '/console/login');
+    it('fails when all credentials are not provided', () => {
+        const expectedErrorMessage = 'Please fill your credentials';
+        const getLoginField = (fieldPlaceholder: string) => {
+            return cy.get(`input[placeholder="${fieldPlaceholder}"]`);
+        };
+        const typeToLoginField = (fieldPlaceholder: string, value: string) => {
+            getLoginField(fieldPlaceholder).clear().type(value);
+        };
+        const submitLoginForm = () => {
+            cy.clickButton('LOGIN');
+        };
+
+        cy.visit('/console/login');
+
+        submitLoginForm();
+        cy.get('.error.message').should('have.text', expectedErrorMessage);
+
+        typeToLoginField('Username', 'test');
+        submitLoginForm();
+        cy.get('.error.message').should('have.text', expectedErrorMessage);
+
+        getLoginField('Username').clear();
+
+        typeToLoginField('Password', 'test');
```

Comment:
Cosmetic readability:
```suggestion
        getLoginField('Username').clear();
        typeToLoginField('Password', 'test');
```